visual-dotnet/SauceLabs.Visual/Models/Browser.cs

```diff
@@ -0,0 +1,11 @@
+namespace SauceLabs.Visual.Models
+{
+    public enum Browser
+    {
+        Chrome,
```

Comment:
enum will be translated into a number. Are you sure the same order is on the server side? Maybe we can set a value to each option to be sure it will always be converted into a correct value (like in the future some adds lets say Internet explorer in the middle of the enum value)

Reply:
Our GraphQL tooling converts these to CONSTANT_CASE to match our API, see [here](https://graphql-dotnet.github.io/docs/getting-started/schema-types/#enumerations).